### ISSUE TITLE
NullPointerException when receiving URL from Pidgin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#41]
```
I tried getting gtalksms to respond in any way to my commands, but its always 
silent. "ring" also does not work. When sending a URL I noticed it disconnects 
and then reconnects. I see this in the log

01-09 12:02:05.104 D/gtalksms( 5030): Xmpp packet received

01-09 12:02:11.264 W/System.err( 5030): java.lang.NullPointerException
01-09 12:02:11.264 W/System.err( 5030): at 
org.jivesoftware.smackx.provider.XHTMLExtensionProvider.parseExtension(XHTMLExte
nsionProvider.java:84)
01-09 12:02:11.264 W/System.err( 5030): at 
org.jivesoftware.smack.util.PacketParserUtils.parsePacketExtension(PacketParserU
tils.java:378)
01-09 12:02:11.264 W/System.err( 5030): at 
org.jivesoftware.smack.util.PacketParserUtils.parseMessage(PacketParserUtils.jav
a:107)
01-09 12:02:11.264 W/System.err( 5030): at 
org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:272)
01-09 12:02:11.264 W/System.err( 5030): at 
org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:44)
01-09 12:02:11.264 W/System.err( 5030): at 
org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:76)
01-09 12:02:11.284 W/gtalksms( 5030): xmpp disconnected due to error: 
java.lang.NullPointerException
01-09 12:02:11.284 V/gtalksms( 5030): broadcasting state transition from 2 to 0
01-09 12:02:11.354 V/gtalksms( 5030): broadcasting state transition from 0 to 4
```

Original issue reported on code.google.com by `sop...@gmail.com` on 9 Jan 2011 at 11:11

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 9 Jan 2011 at 12:58
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 9 Jan 2011 at 1:38
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Tried svn and this works perfectly, thanks. My other problems were 
setup-related.

I do notice though that, when sending a URL, gtalksms (?) asks me what to use 
to handle it. But handling it with itself is not really necessary, right? That 
just echos the URL back on xmpp. If this dialog could be suppressed if the only 
choices are gtalksms and the browser, that would be great.
```

Original comment by `sop...@gmail.com` on 10 Jan 2011 at 7:48

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.7.1
```

Original comment by `Florent....@gmail.com` on 12 Jan 2011 at 11:15
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
